configs/default/FOSS-DRONIUSF7.config

```diff
@@ -0,0 +1,106 @@
+board_name DRONIUSF7
```

Comment:
Now you've removed _two_ lines. I have asked you to remove _one_ line (the second one).